pkg/cmd/server/api/types.go

```diff
@@ -47,10 +47,14 @@ type MasterConfig struct {
 	// CORSAllowedOrigins
 	CORSAllowedOrigins []string
 
+	// MasterPublicURL is how clients can access the OpenShift API server
+	MasterPublicURL string
```

Comment:
> `PublicURL` to match the `AssetConfig`?

I can't decide whether I like everything using the same names to refer to the same things, or removing prefixes based on context. I think I lean toward the same names... but I don't feel that strongly